pyproject.toml

```diff
@@ -86,7 +86,3 @@ markers = [
     "billable: marks test as billable  (deselect with '-m \"not billable\"')",
 ]
 addopts = "-m 'not billable' --ignore=src"
-
-[tool.poetry.scripts]
-create_rai_ws = "rai.cli.rai_cli:create_rai_ws"
-parse_whoami_package = "rai.cli.rai_cli:parse_whoami_package"
```

Comment:
Please revert it or change to sh everywhere

Reply:
already done